packages/gatsby-plugin-nginx/src/nginx-generator.ts

```diff
@@ -275,9 +275,11 @@ function generateProxyRewriteChildren({
   toPath,
   proxyHeaders,
 }: Redirect): NginxDirective['children'] {
+  const isArgs: string = toPath.includes('?') ? '&' : '?'
```

Comment:
We could use `$is_args` instead of `?` but would it even matter if there is a dangling `?`?

Reply:
yeah. I'm not sure either. Let's uses `$is_args` to keep the behavior, what do you think?

Reply:
`$args` would only contain the `a=b&c=d` part. `$is_args` is meant to contain the `?`